lib/trip_dispatcher.rb

```diff
@@ -27,12 +32,51 @@ def inspect
               #{passengers.count} passengers>"
     end
 
+    def available_driver
```

Comment:
This would make an excellent method to make private.

There is one thing, if no driver is available, you simply return `nil`, which means requesting a trip